lib/ipmi_backend_utils.pm

```diff
@@ -34,7 +34,7 @@ sub use_ssh_serial_console {
     console('sol')->disable if check_var('BACKEND', 'ipmi');
     select_console('root-ssh');
     $serialdev = 'sshserial';
-    set_var('SERIALDEV', 'sshserial');
+    set_var('SERIALDEV', $serialdev);
```

Comment:
I think it is subject for another bigger PR, so just for the record - I think all this things like `console('sol')->disable` or setting $serialdev variable should more naturally happen inside `select_console` function.  But nothing related to this certain PR